uast/uast.go

```diff
@@ -377,6 +377,12 @@ const (
 	// annotations.
 	OtherLiteral
 
+	// MapKeyValue is the expression pairing a map key and a value usually on MapLiteral expressions. It must
+	// have both a MapKey and a MapValue children (e.g. {"key": "value", "otherkey": "otherval"} in Python).
+	MapKeyValue
```

Comment:
You are right - I'm bad at naming things that this shows. What about MapEntry? So it will serve in other instances where is used to access keys and values and not only for literals.